osu.Game/Overlays/BeatmapSet/BeatmapSetHeaderContent.cs

```diff
@@ -90,113 +91,117 @@ public BeatmapSetHeaderContent()
                             },
                         },
                     },
-                    new Container
+                    new OsuContextMenuContainer
                     {
-                        RelativeSizeAxes = Axes.X,
-                        AutoSizeAxes = Axes.Y,
-                        Padding = new MarginPadding
-                        {
-                            Vertical = BeatmapSetOverlay.Y_PADDING,
-                            Left = BeatmapSetOverlay.X_PADDING,
-                            Right = BeatmapSetOverlay.X_PADDING + BeatmapSetOverlay.RIGHT_WIDTH,
-                        },
-                        Children = new Drawable[]
+                        RelativeSizeAxes = Axes.Both,
+                        Child = new Container
```

Comment:
This should not use `RelativeSizeAxes = Axes.Both` in this hierarchy, look at the sizing specifications of the parent of this container and the child here.

For reference, this is how this looks now on a beatmap with many difficulties:

| master | PR |
|--------|-------|
| <img width="1708" alt="CleanShot 2022-07-26 at 10 42 49@2x" src="https://user-images.githubusercontent.com/22781491/180951519-824174e7-fd8b-4870-aff8-49c51f63f3a3.png"> | <img width="1708" alt="CleanShot 2022-07-26 at 10 41 17@2x" src="https://user-images.githubusercontent.com/22781491/180951201-85ba56c4-ebad-4f9a-a066-7404b624bc7c.png"> |